src/Microsoft.AspNetCore.Hosting/project.json

```diff
@@ -50,7 +52,7 @@
         }
       }
     },
-    "netstandard1.3": {
+    "netstandard1.5": {
```

Comment:
Also anyone taking an RC2 dependency on aspnetcore libs will have to bump from `netstandard1.3` to `netstandard1.5` for rtm